spec/facebook_ads/model_spec.rb

```diff
@@ -0,0 +1,220 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+describe FacebookAds::Model do
+  let(:test_class) do
+    Class.new(described_class) do
+      field :foo
+      field :bar
+    end
+  end
+
+  let(:id) { 42 }
+  let(:test_model) { test_class.new(id) }
+
+  let(:client) { double(FacebookAds::Client) }
+
+  before do
+    allow(FacebookAds).to receive(:client).and_return(client)
+  end
+
+  describe '.field' do
+    it 'creates a round-tripping accessor' do
```

Comment:
Roundtrip refers to the fact, that after calling the setter, I get the expected value from the getter... (and not something else).

But I will simplify my words ^^